velox/dwio/parquet/reader/ParquetTypeWithId.cpp

```diff
@@ -86,15 +90,14 @@ bool ParquetTypeWithId::hasNonRepeatedLeaf() const {
 }
 
 LevelMode ParquetTypeWithId::makeLevelInfo(LevelInfo& info) const {
-  int16_t repeatedAncestor = 0;
-  for (auto parent = parquetParent(); parent;
-       parent = parent->parquetParent()) {
-    if (parent->type()->kind() == TypeKind::ARRAY ||
-        parent->type()->kind() == TypeKind::MAP) {
-      repeatedAncestor = parent->maxDefine_;
-      break;
+  int16_t repeatedAncestor = maxDefine_;
```

Comment:
Seems this variable is not used anywhere

Reply:
Its put in level info and then its used in arrow internally in arrow calls[DefLevelsToBitmap](https://github.com/facebookincubator/velox/blob/7fc09667d5e22c684fdeff81da529b79cc974fee/velox/dwio/parquet/reader/PageReader.cpp#L573).

Reply:
Ah I see.  Lint is complaining about the type narrowing here, let's fix it to `int` here

Reply:
Okay were the crashes fixed?

Reply:
Yes it seems fixed